source/includes/_pis.md

```diff
@@ -104,3 +97,7 @@ we don't let you include both.
 At the moment, we don't support the `payment-details` endpoint.
 
 You can only make payments in `GBP`. We don't support other currencies.
+
+## Testing in the Sandbox
+
+In the **sandbox** environment, you can automatically have domestic payment requests approved or declined to help with testing. When creating the payment consent, you can add a `DesiredStatus` field to the `Data/Initiation/SupplementaryData` object in the consent request. You can set this field to `Authorised` or `Rejected`, depending on the behaviour you want.
```

Comment:
I think we need to ask them to supply user + account information as well.

Reply:
It's optional, and I don't think it'll make any difference (as long as we keep that test account funded!)